icgcget/cli.py

```diff
@@ -18,43 +18,27 @@
 
 import logging
 import os
-
+from tabulate import tabulate
 import click
-import yaml
+
+import psutil
+from base64 import b64decode
 
 from clients.ega import ega_client
 from clients.gdc import gdc_client
 from clients.gnos import gt_client
+from clients.icgc import icgc_api
 from clients.icgc import icgc_client
-from utils import flatten_dict, normalize_keys
+from utils import file_size, config_parse, match_repositories
 
-REPOS = ['collab', 'aws', 'ega', 'gdc', 'cghub']
+REPOS = ['collaboratory', 'aws-virginia', 'ega', 'gdc', 'cghub']  # Updated for codes used by api
 
 DEFAULT_CONFIG_FILE = os.path.join(click.get_app_dir('icgc-get', force_posix=True), 'config.yaml')
-
-
-def config_parse(filename):
-    config = {}
-    try:
-        config_text = open(filename, 'r')
-    except IOError:
-
-        print("Config file {} not found".format(filename))
-        return config
-
-    try:
-        config_temp = yaml.safe_load(config_text)
-        config_download = flatten_dict(normalize_keys(config_temp))
-        config = {'download': config_download, 'logfile': config_temp['logfile']}
-    except yaml.YAMLError:
-
-        print("Could not read config file {}".format(filename))
-        return {}
-
-    return config
+global logger
 
 
 def logger_setup(logfile):
+    global logger
```

Comment:
Some of them.  It's much easier to do here because I know for certain where the logger will be called first.